test/tests/functional/pbs_cray_smoketest.py

```diff
@@ -118,27 +127,22 @@ def test_cray_login_job(self):
 
         output_file = os.path.join(
             sub_dir, 'cray_login.o' + jid1.split('.')[0])
-        found = 0
         with open(output_file, 'r') as of:
             for line in of:
                 if "Hello World" in line:
-                    found = 1
+                    self.assertTrue("Hello World" in line,
+                                    msg="Job output file incorrect")
```

Comment:
@vccardenas in your prior commit (f87161f) you had replaced the assertTrue for Hello World with the "found" and the asserEqual on found.  With the comment "corrections to parsing of job output file".
But now, you've reverted those changes.  I'm not sure I understand why the code is changing, can you please help me understand?

Reply:
That was a mistake.  I have corrected that and moved the finding of the string "Hello World" into a function.